spec/models/office_spec.rb

```diff
@@ -77,10 +99,19 @@
     end
 
     context 'when provided' do
-      before { office.telephone_number = ' 07715 930 457  ' }
+      it 'removes whitespace from the front and back' do
```

Comment:
Given that this is being done `#telephone_number=` I'm not sure that this is actually testing what you are intending it to test. If you set the telephone number without using the setter, then the getter's behaviour can be tested.